test/controllers/JourneyAnswersControllerSpec.scala

```diff
@@ -131,7 +132,8 @@ class JourneyAnswersControllerSpec extends ControllerBehaviours with ScalaCheckP
         abroadAnswersService = StubAbroadAnswersService(getAnswersRes = Some(answers).asRight),
         incomeService = StubIncomeAnswersService(),
         expensesService = StubExpensesAnswersService(),
-        capitalAllowancesService = StubCapitalAllowancesAnswersAnswersService()
+        capitalAllowancesService = StubCapitalAllowancesAnswersAnswersService(),
+        prepopAnswersService = StubPrepopAnswersService()
```

Comment:
In my next PR I'll make these into a common val for the empty stubbed Controller to be used by all of these tests

Reply:
This didn't end up being a viable solution, have instead replaced many of them with an existing test method that covers the same bases.